### PR TITLE
ref(onboarding) Update project platform to typescript, and clean up the design

### PR DESCRIPTION
**Pull request by markstory**
* Convert to typescript
* Clean up design

#### Desktop
Screen Shot 2020-11-10 at 3 27 58 PM

#### Mobile
Screen Shot 2020-11-10 at 3 33 45 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectInstall/platform.tsx [hunk 1]
```diff
@@ -87,35 +98,36 @@ class ProjectInstallPlatform extends React.Component {
...
+    const platformLink = platform.link ?? undefined;
 
     return (
-      <Panel>
-        <PanelHeader hasButtons>
-          {t('Configure %(platform)s', {platform: platform.name})}
-          <Actions>
+      <React.Fragment>
+        <StyledPageHeader>
+          <h2>{t('Configure %(platform)s', {platform: platform.name})}</h2>
```
Comment (matejminar) [inline src/sentry/static/sentry/app/views/projectInstall/platform.tsx:106]: What's the difference between this and `tct`?

**Review by matejminar — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectInstall/platform.tsx (hunk 1 above)
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/projectInstall/platform.tsx:106]: shouldn't it be only`platform`?

**Review by priscilawebdev — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectInstall/platform.tsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/projectInstall/platform.tsx:106]: tct() can format nodes as well.

Comment (markstory) [reply src/sentry/static/sentry/app/views/projectInstall/platform.tsx:106]: It is platform. The trailing `s` is to indicate the it is a string format.

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/projectInstall/platform.tsx:106]: interesting :)

**Pull request merged**